docs/en/04_Changelogs/4.10.0.md

```diff
@@ -1,19 +1,119 @@
+| silverstripe/ckan-registry | 1.4.0 |
+| silverstripe/comment-notifications | 2.2.0 |
+| silverstripe/comments | 3.7.2 |
+| silverstripe/content-widget | 2.3.0 |
+| silverstripe/contentreview | 4.4.0 |
+| silverstripe/crontask | 2.4.0 |
+| silverstripe/documentconverter | 2.2.0 |
+| silverstripe/elemental-bannerblock | 2.4.0 |
+| silverstripe/elemental-fileblock | 2.4.0 |
+| silverstripe/environmentcheck | 2.4.0 |
+| silverstripe/externallinks | 2.2.0 |
+| silverstripe/fulltextsearch | 3.9.0 |
+| silverstripe/gridfieldqueuedexport | 2.6.1 |
+| silverstripe/html5 | 2.2.0 |
+| silverstripe/hybridsessions | 2.4.0 |
+| silverstripe/iframe | 2.2.0 |
+| silverstripe/ldap | 1.3.0 |
+| silverstripe/mfa | 4.5.1 |
+| silverstripe/realme | 4.2.0 |
+| silverstripe/registry | 2.4.0 |
+| silverstripe/restfulserver | 2.4.0 |
+| silverstripe/security-extensions | 4.2.1 |
+| silverstripe/securityreport | 2.4.0 |
+| silverstripe/segment-field | 2.4.0 |
+| silverstripe/sharedraftcontent | 2.6.0 |
+| silverstripe/sitewidecontent-report | 3.2.0 |
+| silverstripe/spamprotection | 3.2.0 |
+| silverstripe/spellcheck | 2.3.0 |
+| silverstripe/subsites | 2.5.0 |
+| silverstripe/tagfield | 2.8.0 |
+| silverstripe/taxonomy | 2.3.0 |
+| silverstripe/textextraction | 3.3.0 |
+| silverstripe/totp-authenticator | 4.3.0 |
+| silverstripe/userforms | 5.11.1 |
+| silverstripe/versionfeed | 2.2.0 |
+| silverstripe/webauthn-authenticator | 4.4.1 |
+| silverstripe/widgets | 2.2.0 |
+| symbiote/silverstripe-advancedworkflow | 5.6.0 |
+| symbiote/silverstripe-multivaluefield | 5.2.0 |
+| symbiote/silverstripe-queuedjobs | 4.9.0 |
+| tractorcow/silverstripe-fluent | 4.6.0 |
+
+</details>
+
 
 ## Regression test and Security audit{#audit}
 
 This release has been comprehensively regression tested and passed to a third party for a security-focused audit.
 
 While it is still advised that you perform your own due diligence when upgrading your project, this work is performed to ensure a safe and secure upgrade with each recipe release.
 
+## PHPUnit 9.5 and PHP 8.0 official support{#php8}
+
+Previous Silverstripe CMS Recipe releases could be run against PHP 8.0. However, the Silverstripe CMS Recipe 4.10.0 is the first release to officially support PHP 8.0. 
+
+The major road block to official PHP 8.0 support was our reliance on a deprecated version of PHPUnit. Adding support for PHPUnit 9.5 removed this road block. All core module tests have been rewritten to run with PHPUnit 9.5.
+
+### What about my project test suite?
+
+Silverstripe CMS Recipe 4.10.0 retains support for the `sminnee/phpunit` fork of PHPUnit 5.7. So your project's tests can still be run without modification ... as long as you are not using PHP 8.0. If you want your own tests to run with PHP 8.0, then you have have to upgrade to PHPUnit 9.5 and update your tests to be compatible with the latest PHPUnit API.
+
+We'll be providing guidance on how to transition to PHPUnit 9.5 closer to the release.
```

Comment:
Is now the "closer to the release" time? Should it read a bit differently, e.g. "close after the 4.10" release?

Reply:
Just provided a link to said guidance.